release-notes/aws/v10.1.0.md

```diff
@@ -0,0 +1,37 @@
+# :zap: Giant Swarm Release 10.1.0 for AWS is now active for you! :zap:
+
+We are pleased to announce that the new Cluster API enabled release v10.1.0 contains the much anticipated Node Pools feature.
+
+We want to remind you that the Node Pools enabled releases imply a breaking change. This means **existing clusters** below the 10.x.x major releases **are not upgradable**. 
+We will continue to support the 9.x.x releases with bug fixes and security patches for the next 6 months. During that timeframe please **ensure you  move all your workloads to new clusters**. We are at your service to assist with this.
+
+*Please note*: With this release, we are NO LONGER rolling out NGINX Ingress Controller by default. It is now an optional App in the App Catalog and can be installed and configured on-demand. Or you can choose your own.
```

Comment:
I am sorry for this last minute notice, but we just realized that optional IC is blocked in China for the lack of route53. We have a plan to fix this, but for now we decided to go with a notice that with 10.1.0 the optional IC will not be available in China and we would recommend that customers in China wait for our fixes for China to land in a soon to be released patch version before they start clusters with 10.x. cc @teemow you fine with that?

Reply:
Is this resolved with the latest addition https://github.com/giantswarm/releases/pull/45/commits/72f7e50f4225f0c9e58c2fac5270794ccbe69beb ?